pyprof/prof/index_slice_join_mutate.py

```diff
@@ -61,23 +61,14 @@ def tc(self):
         return "-"
 
     def op(self):
-        return self.op_
+        return self._op
 
     def mod(self):
-        return self.mod_
+        return self._mod
 
     def bytes(self):
-        b = 0
-        for s in self.shapes:
-            b += Utility.numElems(s)
-
-        b = 2 * b * Utility.typeToBytes(self.type)
-
-        if (self.sub == 0):
-            return b
-        else:
-            return 0
-
+        b = 2 * reduce(operator.add, [t.bytes for t in self.inp])
```

Comment:
What does the reduce method do?

Reply:
`reduce` is a python builtin for the reduction operation.